### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by tobydarling** [45Drives/cephgeorep#37] [General]
Hi Josh, this is great stuff! We've got about 10PB of ceph, roughly 50% of it on 45Drives systems backing up the primary. I've knitted a bunch of shell+perl to try and achieve what cephgeorep does, but mine's nothing like as sophisticated.
I hope to be able to contribute (once I've either got a build env). You use the github issue tracker?
Cheers
Toby

**Comment by joshuaboud** [comment]
Hi Toby, I'm glad to hear you like it! We do not currently use Github Issue Tracker, however if you find any issues you'd like to report, feel free to open a new issue in the issues tab as I check that regularly.
